jinja2/includes/footer/moz-footer.html

```diff
@@ -0,0 +1,25 @@
+<a href="https://mozilla.org" class="nav-footer-mozilla">{{ _('Mozilla') }}</a>
+<div class="footer-group footer-group-mozilla">
+  <h2 class="footer-title">Mozilla</h2>
+  <ul class="footer-list">
+    <li><a href="https://www.mozilla.org/about/">{{ _('About') }}</a></li>
+    <li><a href="https://www.mozilla.org/contact/">{{ _('Contact Us') }}</a></li>
+    <li><a href="https://donate.mozilla.org/?utm_source=developer.mozilla.org&utm_medium=referral&utm_content=footer">{{ _('Donate') }}</a></li>
+    <li><a href="https://www.mozilla.org/firefox/?utm_source=developer.mozilla.org&utm_campaign=footer&utm_medium=referral">Firefox</a></li>
+    <li class="footer-social">
+        <a href="https://twitter.com/mozilla">
+            {% include 'includes/icons/social/github.svg' %}
```

Comment:
This changes from a span with a replaced name to an SVG icon. We may need to think about [accessible SVG](https://www.sitepoint.com/tips-accessible-svg/), with (translated) titles and descriptions.

Reply:
Yup, I was planning on adding titles where it makes sense. I was just not sure of the process of introducing new strings. For things like Twitter, Facebook etc. it is simple. I will have a follow up to resolve these. Thanks for pointing it out, and thinking of it.